_source/_posts/2022-02-25-thymeleaf-security.md

```diff
@@ -0,0 +1,670 @@
+---
+layout: blog_post
+title: "Securing your application when using Thymeleaf templates with Spring Webflux"
+author: jimena-garbarino
+by: contractor
+communities: [security,java]
```

Comment:
```suggestion
communities: [java]
```
It seems that most of our tutorials could be security and X, so I think it's best to just do X.